### ISSUE TITLE
the Instance normalization  is very very slow........

### ISSUE THREAD

**Issue by nextfeixiag** [KupynOrest/DeblurGAN#68]
it takes a  lot of time...

**Comment by KupynOrest** [comment]
You can try to train the model with batch norm & batchSize=1 to achieve the same result, it might speed up the inference if you need
